src/public/interfaces.ts

```diff
@@ -394,3 +394,10 @@ export interface OpenConversationRequest {
   */
   onCloseConversation?: (subEntityId: string, conversationId?: string) => void;
 }
+
+export interface BotAPIRequest {
```

Comment:
Could we name this BotRequestMessage (echoing .NET's HttpRequestMessage or Node.js http.IncomingMessage)
The "API" part feels redundant, and a bit in-apt, as it's a pretty generic mechanism, not a specific API.

Reply:
BotRequest?